database.js

```diff
@@ -0,0 +1,18 @@
+const mongoose = require('mongoose');
+
+let mongoServer;
+const uri="mongodb+srv://sckchcm:17%40$aksham@cluster0.rpxpqex.mongodb.net/?retryWrites=true&w=majority"
```

Comment:
Avoid hardcoding the MongoDB URI. Instead, store it in an environment variable. This prevents sensitive information from being exposed in the code. [important]